### PR TITLE
Feature/monitoring stack ver1

### PR DESCRIPTION
**Pull request by natmasslove**

### CODE REVIEW

File: infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        get_common_stack_references(): Retrieves references to artifacts created in common stack (like S3 bucket, SNS topic, ...)
+        get_metrics_collection_interval_min(): Fetches the interval for metrics collection from general.json.
+        create_extract_metrics_lambdas(settings_bucket, internal_error_topic, timestream_database_arn):
+            Creates Lambda functions for extracting metrics.
+    """
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        self.stage_name = kwargs.pop("stage_name", None)
+        self.project_name = kwargs.pop("project_name", None)
+
```
Comment (NastassiaPaskrobka) [inline infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py:36]: Please add doc-string for constructor

File: infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+            general_config = json.load(f)
+            tooling_section = general_config["tooling_environment"]
+            key = "metrics_collection_interval_min"
+            if key in tooling_section:
+                return tooling_section[key]
+            else:
+                raise Exception(
+                    "metrics_collection_interval_min key not found in general.json config file ('tooling_environment' section)"
+                )
+
```
Comment (NastassiaPaskrobka) [inline infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py:124]: Usually user-defined exception is used instead, but most probably all this settings part will be rewritten soon :)

**Review by NastassiaPaskrobka — APPROVED** [review]

File: infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py (hunk 1 above)
Comment (natmasslove) [reply infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py]: fixed :)

File: infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py (hunk 2 above)
Comment (natmasslove) [reply infra_tooling_account/infra_tooling_account/infra_tooling_monitoring_stack.py:135]: yes, it's just temporary

**Review by vdektiarev — APPROVED** [review]
LGTM

**Pull request merged**